OrbitCore/OrbitProcess.h

```diff
@@ -31,10 +31,10 @@ struct ModuleDebugInfo;
 struct IDiaSymbol;
 
 //-----------------------------------------------------------------------------
-class Process {
+class Process final {
```

Comment:
Why is "final" needed here?

Reply:
It is not needed, just wanted to make sure nobody inherits this class and therefore the protected: below could safely be changed to private:

Reply:
"protected" served no purpose, I agree, let's just private.  Having "final" here with no explanation is confusing.  What would happen we derived a class from Process, would anything break?  Process is not part of a hierarchy anyway, I'd remove "final", or comment on why it is there.

Reply:
removed